lib/api/reports.js

```diff
@@ -0,0 +1,77 @@
+export const reportBug = ({
+    OS,
+    OSVersion,
+    Browser,
+    BrowserVersion,
+    BrowserExtensions,
+    Resolution,
+    DisplayMode,
+    Client,
+    ClientVersion,
+    ClientType,
+    Title,
+    Description,
+    Username,
+    Email,
+    Country,
+    ISP,
+}) => ({
+    method: 'post',
+    url: 'reports/bug',
+    data: {
+        OS,
+        OSVersion,
+        Browser,
+        BrowserVersion,
+        BrowserExtensions,
+        Resolution,
+        DisplayMode,
+        Client,
+        ClientVersion,
+        ClientType,
+        Title,
+        Description,
+        Username,
+        Email,
+        Country,
+        ISP,
+    },
+});
+
+export const reportCrash = ({
+    OS,
+    OSVersion,
+    Browser,
+    BrowserVersion,
+    Client,
+    ClientVersion,
+    ClientType,
+    Debug,
+  }) => ({
+    method: 'post',
+    url: 'reports/bug',
+    data: {
+        OS,
+        OSVersion,
+        Browser,
+        BrowserVersion,
+        Client,
+        ClientVersion,
+        ClientType,
+        Debug,
+    },
+});
+
+// reportCrashSentryProxy  unclear API-spec
+
+export const reportCSPViolation = (cspReport) => ({
```

Comment:
This one is weird, check with @bartbutler please.

Reply:
Should I ask him why is it in the api-spec at all? Or the weird thing is in the details of the api call?

Reply:
Why the variable is not Capitalized.

Reply:
Again quoting @bartbutler : 

> In general, all parameters in the API are PascalCase
> The exceptions are all implementations of APIs that are standards that aren’t ours
> the CSP report one is one such API, see here: https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP